pkg/didcomm/msghandler/msghandler.go

```diff
@@ -12,13 +12,15 @@ package msghandler
 import (
 	"fmt"
 	"sync"
+	"sync/atomic"
 
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
 )
 
 const (
 	errAlreadyRegistered = "registration failed, message service with name `%s` already registered"
 	errNeverRegistered   = "failed to unregister, unable to find registered message service with name `%s`"
+	outboundAlreadySet   = "outbound is already set"
```

Comment:
missing `err` prefix (like the two above)

Reply:
fixed